geoengine_partner/views/partner.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record model="ir.ui.view" id="geo_partner_view_form">
+        <field name="name">geo_partner_form</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="arch" type="xml">
+            <notebook>
+                <page
+                    string="Geoengine Map"
+                    name="geoengine_map"
+                    groups="base_geoengine.group_geoengine_user,base_geoengine.group_geoengine_admin"
+                >
+                    <field name="geo_point" />
+                </page>
+            </notebook>
+        </field>
+    </record>
+
+    <record id="ir_ui_view_geo_partner" model="ir.ui.view">
+        <field name="name">Partner view</field>
+        <field name="arch" type="xml">
+            <geoengine>
```

Comment:
Here, if we want to make the view editable, we need to add the attribute _editable_ for example : `<geoengine editable="1">`